src/helper/task.ts

```diff
@@ -351,6 +351,24 @@ export class ZkWasmServiceHelper {
     return tasks;
   }
 
+  async getTaskExternalHostTable(
```

Comment:
Need refine the example/src/query/task.ts queryTasks() example, also add a new example about how to get the external host table file by new method